timescale-cloud/index.md

```diff
@@ -1,38 +1,74 @@
 # Timescale Cloud
+Managed services for TimescaleDB (MST) provides managed TimescaleDB services
+hosted in more than 75 regions using AWS, Azure, or GCP. You can use MST to
+create database instances, or services, in the cloud and automate most of your
+most common operational tasks. This allows you to spend more time focusing on
+your time-series workloads and less time worrying about database management.
 
-Timescale Cloud is a managed TimescaleDB service hosted in 75+ regions in AWS, 
-Azure, or GCP. You can [try Timescale Cloud for free][sign-up], no credit card 
-required.
+*   Find out [about services][create-a-service]
+*   Create a [multi-node cluster][cloud-multi-node]
+*   View [service logs][viewing-service-logs]
+*   Use [VPC peering][vpc-peering]
+*   Understand [security for MST][security]
 
-### Hosted TimescaleDB
-Built and operated by the team behind TimescaleDB, Timescale Cloud provides a 
+<highlight type="note">
+Managed service for TimescaleDB (MST) used to be called _Timescale Cloud_.
+</highlight>
+
+Some important MST terms:
+*   Account: Your MST account. You can register for an account on the
+    [MST signup][sign-up] page.
+*   Project: An empty project is created for you automatically when you sign up.
+    Projects help you organize your services and billing settings. You can also
+    use projects to manage user access to your services.
+*   Service: A service is an instance that corresponds to a cloud service
+    provider tier. You can access all your services from the `Services` tab
+    within your projects.
+*   Database: Databases are created within a service. You can view and create a
+    database within a service by selecting one of your services, and then
+    navigating to the `Databases` tab.
+*   Service plans: A service plan defines the configuration and level of
```

Comment:
```suggestion
*   **Project:** An empty project is created for you automatically when you sign up.
    Projects help you organize your services and billing settings. You can also
    use projects to manage user access to your services.
*   **Service:** A service is an instance that corresponds to a cloud service
    provider tier. You can access all your services from the `Services` tab
    within your projects.
*   **Database:** Databases are created within a service. You can view and create a
    database within a service by selecting one of your services, and then
    navigating to the `Databases` tab.
*   **Service plans:** A service plan defines the configuration and level of
```

Just a suggestion. I like when the terms are bolded so that I can separate them in lots of text.

Reply:
Yeah, that's fair enough. It would be great if we could do proper deflists in markdown.